data/i18n/en/en.toml

```diff
@@ -160,24 +160,96 @@ other = "Tell your story"
 [layouts_docs_glossary_aka]
 other = "Also known as"
 
-[layouts_docs_glossary_description]
-other = "This glossary is intended to be a comprehensive, standardized list of Kubernetes terminology. It includes technical terms that are specific to Kubernetes, as well as more general terms that provide useful context."
+[layout_docs_glossary_architecture_name]
+other = "Architecture"
 
-[layouts_docs_glossary_deselect_all]
-other = "Deselect all"
+[layout_docs_glossary_architecture_description]
+other = "The inner components of Kubernetes."
 
 [layouts_docs_glossary_click_details_after]
 other = "indicators below to get a longer explanation for any particular term."
 
 [layouts_docs_glossary_click_details_before]
 other = "Click on the"
 
+[layout_docs_glossary_community_name]
+other = "Community"
+
+[layout_docs_glossary_community_description]
+other = "Related to Kubernetes open-source development."
+
+[layout_docs_glossary_core-object_name]
+other = "Core Object"
```

Comment:
Hello @shubhankar-mern .
How are you ordering the fields?
For example, should `layout_docs_glossary_core-object_name` appear before `layout_docs_glossary_core-object_description`

Reply:
Object description comes before object name .....d before n @kbhawkey